core/nodejsActionBase/Dockerfile

```diff
@@ -15,17 +15,30 @@
 # limitations under the License.
 #
 
-FROM buildpack-deps:trusty-curl
-
-ENV DEBIAN_FRONTEND noninteractive
+FROM node:stretch
```

Comment:
Are we ok with this floating to whatever the latest node version is, or should we bind it a little tighter?

Reply:
we don't actually build or publish this image it's primarily used as a template. So I was thinking this is ok in that context.